### DISCUSSION TITLE
Add more informative logs for database connection problem

### DISCUSSION THREAD

**Discussion by dmitrygribenchuk** [directus/directus#16613] [Archive]
Hello. When I deploy Directus for the first time I made an error with the database configuration (wrong password). In container logs were only "Initializing bootstrap..." every time when container was restarted. Is it possible to add some additional information about database connection problem? 

![image](https://user-images.githubusercontent.com/15670247/203934549-5e37544f-2266-4428-b3a3-c15ffb13d214.png)

### What version of Directus are you using?

9.21.0

### What version of Node.js are you using?

16.18.1

### What database are you using?

Postgres

### How are you deploying Directus?

Docker

**Comment by paescuj** [comment]
This is a known issue, see #16284.

As @azrikahar pointed out, you can use `LOG_STYLE=raw` to surface all logs.
